docker/up.sh

```diff
@@ -47,6 +47,7 @@ usage() {
   title "FLAGS:"
   echo "  -b, --build           build images from source"
   echo "  -s, --seed            seed HTTP API server with metadata"
+  echo "  -d, --daemon          run in the background"
```

Comment:
Mind if we use `--detach` instead to match the docker option for running containers in [detached mode](https://docs.docker.com/engine/reference/run/#detached--d)?

Reply:
Ah, and here I always thought -d stood for "daemon" 😀

I'll update it.